src/components/Footer/Footer.jsx

```diff
@@ -14,88 +14,162 @@ export const Footer = ({
   img = "https://generation-sessions.s3.amazonaws.com/27ef0faa38aabce506c661f34f7307c9/img/--2.svg",
 }) => {
   return (
-    <div className={`footer ${className}`}>
-      <div className="frame">
-        <div className="div">
-          <p className="text-wrapper">Not sure where to start?</p>
-          <p className="request-a-demo-call">
-            Request a demo call and get all your queries answered. <br />
-            Don’t Worry, we will not trigger spam calls.
-          </p>
-        </div>
-        <div className="frame-2">
-          <div className="overlap-group-wrapper">
-            <div className="overlap-group">
-              <div className="text-wrapper-2">Join</div>
-            </div>
+
+    <footer class="footer_color">
+      <div class="flex flex-col space-y-20">
+        <div class="flex flex-col space-y-8 lg:flex-row lg:space-x-40 xl:flex-row xl:space-x-70 lg:items-center py-10 px-16 lg:pl-10">
+          <div class="flex flex-col space-y-4 px-8 pt-8">
+            <span class="text-xl lg:text-4xl text-white font_monasans w-auto">Not sure where to start?</span>
+            <p class="text-sm lg:text-sm max-w-[400px] text-white font_monasans">Request a demo call and get all your queries answered. Don’t Worry, we will not trigger spam calls.</p>
+          </div>
+          <div class="flex xl:pl-60">
+            <input type="email" id="email" class="enter_mail input_text input" placeholder="Enter email" required />
+            <button class="signup_color rounded-r-full px-8 my-0 text-white text-size-2">Join</button>
           </div>
-          <div className="text-wrapper-3">Enter Email</div>
         </div>
-      </div>
-      <img
-        className="line"
-        alt="Line"
-        src="https://generation-sessions.s3.amazonaws.com/27ef0faa38aabce506c661f34f7307c9/img/line-1-1.svg"
-      />
-      <div className="frame-3">
-        <div className="frame-4">
-          <div className="frame-5">
-            <div className="group">
-              <div className="img-wrapper">
-                <img className="img" alt="Img" src={img} />
+
+        <div class="flex flex-col lg:flex-row px-8 py-10">
+          <div class="flex flex-col lg:w-1/3 px-2">
+            <a href="/">
+              <div class="flex items-center">
+
+                <img className="px-2" alt="Img" src="" />
```

Comment:
[Doubt]
Why we are having here empty `img` tag?

Reply:
I forgot about it. I have added the img tag now.
And yes the grid is working. Its not working in my device but Gagan checked its working.

Reply:
@himanshush200599 Do you want screenshots of home page or everyscreen?

Reply:
The screenshot of UI's Screen for different screen-type  less than 500px, 760px, 1080px and one bigger 
@palak-chaturvedi